mateuszbuda_brain-segmentation-pytorch_unet.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: hub_detail
+background-class: hub-background
+body-class: hub
+category: researchers
+title: U-Net
```

Comment:
i think the title should be more specific. "U-Net for Medical" or something, dont you think?
I say this because U-Net is pretty generic, can be used for a lot of things and a baseline implementation should be in torchvision or something (cc: @fmassa )

Reply:
I agree with @soumith that having a title "U-Net for brain MRI" or related might be better.
Even though U-Net original paper was trained on medical data, the [dataset was different I believe](http://brainiac2.mit.edu/isbi_challenge/), so making this distinction might be good.

Also, we should have a reference U-Net in torchvision as well, as pointed out by @soumith , but I'd initially like to have it trained on COCO/Pascal, so I'd need to see if there are published papers that apply U-Net for those datasets (there probably is, I just don't know of them yet :-) )

Reply:
I also agree. Title changed to "U-Net for brain MRI".